numpy/__init__.pyi

```diff
@@ -1235,3 +1235,7 @@ def amin(
     initial: _NumberLike = ...,
     where: _ArrayLikeBool = ...,
 ) -> Union[number, ndarray]: ...
+def rollingmean(
+    a: _ArrayLike,
```

Comment:
Is the underscore intentional?

Reply:
Oh! Good catch, it was unintentional. Will fix. :)